src/components/Code/Code.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { isFunction } from 'lodash';
+
+const CopyNotice = styled.div`
+  position: absolute;
+  top: 5px;
+  right: 5px;
+  transition: opacity 300ms ease;
+  opacity: 0;
+  font-size: ${props => props.theme.fontSizes.tiny};
+  color: ${props => props.theme.colors.athensGray};
+`;
+
+const Root = styled.div`
```

Comment:
`Root` is an odd name for an element (it gets shown in `className`) - normally we'd name this one `CodeWrapper` or something similar :)